test/functional/email_confirmations_controller_test.rb

```diff
@@ -130,6 +130,91 @@ class EmailConfirmationsControllerTest < ActionController::TestCase
     end
   end
 
+  context "on POST to webauthn_update" do
+    context "with webauthn enabled" do
+      setup do
+        @user = create(:user)
+        @webauthn_credential = create(:webauthn_credential, user: @user)
+        get :update, params: { token: @user.confirmation_token, user_id: @user.id }
+        @challenge = session[:webauthn_authentication]["challenge"]
+        @origin = "http://localhost:3000"
+        @rp_id = URI.parse(@origin).host
+        @client = WebAuthn::FakeClient.new(@origin, encoding: false)
+        WebauthnHelpers.create_credential(
+          webauthn_credential: @webauthn_credential,
+          client: @client
+        )
+        post(
+          :webauthn_update,
+          params: { user_id: @user.id,
+            token: @user.confirmation_token,
+            credentials:
+            WebauthnHelpers.get_result(
+              client: @client,
+              challenge: @challenge
+            ) }
```

Comment:
nit: the `}` should be in the next line (there's multiple areas where it's formatted that way)

Reply:
Unfortunately linting fails if it isn't like this :(